components/constellation/constellation.rs

```diff
@@ -818,6 +818,18 @@ where
         namespace_id
     }
 
+    /// Pre-generate a buffer of namespace ids, so content, or other, processes,
+    /// do not require to ipc for them initially.
```

Comment:
I imagine that any IPC for pipeline namespace generation will be completely dwarfed by the time it takes for the network request to complete. I don't think this is worth including without data backing it up.